openslides_backend/action/action.py

```diff
@@ -77,7 +77,7 @@ def __init__(
     ) -> None:
         self.services = services
         self.permission = services.permission()
-        self.datastore = services.datastore()
+        self.datastore = relation_manager.datastore
```

Comment:
Formerly we initiate the datastore object like we do with permission object, auth object and media object. Now this is not done (in the Relation Manager it is not done). Why?

Reply:
The instance of the datastore tracks the locked_fields. Therefore it is necessary to use only one instance during the handling of a request. This instance is created in the request-handler, explicitly in BaseHandler and should be "distributed" to the dependent classes like the Action class or RelationManager.

Reply:
Ok, I understand. In #358 I propose a more explicit way.